.github/workflows/deploy-main.yml

```
@@ -81,5 +81,9 @@ jobs:
         env:
           BRANCH: ${{needs.detect.outputs.branch}}
 
+      - name: update CSS and other assets
+        if: github.ref == 'refs/heads/develop'
+        run: cp -r .github/assets admin-interface-test/
```

Github won't let me make suggestions past line 70, so I'm going to put it here instead.  Insert this at 68 (before the `git checkout`):
```
cp -r .github/assets assets_temp
cp src/image/opencast-white.svg assets_temp
```

This will save the relevant asset folder somewhere that the `git checkout` won't touch.  We know that we saved the assets into a safe place earlier, now we just tell `git` to pay attention to those files.  If the files are the same this ends up doing nothing, but if they're *not* then we end up with whatever is current in `develop`

```suggestion
        run: |
          rm -rf assets
          mv assets_temp assets
          git add assets
          git diff --staged --quiet || git commit --amend -m "Build $(date)"
```